src/main/java/com/github/stephengold/joltjni/AaBox.java

```diff
@@ -238,6 +293,22 @@ public boolean contains(Vec3Arg point) {
         return result;
     }
 
+    /**
+     * Check if the box contains the other box. The box is not affected.
+     *
+     * @param other The other box to check
+     *
+     * @return {@code true} if contained, otherwise {@code false}
+     */
+    @Override
+    public boolean contains(ConstAaBox other) {
```

Comment:
contains(ConstAaBox) should come before contains(Vec3Arg) because C comes before V in the alphabet